src/js/backend/BlockchainLink.js

```diff
@@ -112,23 +109,12 @@ export default class Blockchain {
         }
     }
 
-    async loadTransaction(id: string) {
-        const transaction = await this.link.getTransaction(id);
-        return BitcoinJsTransaction.fromHex(transaction.tx.hex, this.coinInfo.network);
-    }
-
```

Comment:
this is just some cleanup of unused methods right?

Reply:
right, BitcoinJsTransaction logic is moved to `refTxs` utility.
its not need here, in blockchainlink file